lib/3scale/backend/logging/external/impl/bugsnag.rb

```diff
@@ -56,7 +54,7 @@ def configure(api_key)
                   config.timeout = 3
                   config.logger = Backend.logger
                   config.meta_data_filters = []
-                  config.notify_release_stages = ['production', 'preview']
+                  config.notify_release_stages = ['production', 'staging', 'development']
```

Comment:
Also, I'd probably suggest something like 

```
config.enabled_release_stages = ENV['BUGSNAG_ENABLED_STAGES'] || %w[production staging]
```

And then in development we can set `BUGSNAG_ENABLED_STAGES=development` if we really need to, and avoid spamming Bugsnag from development environment when we've got some WIP that throws exceptions all the time :grimacing: 

This would also allow us to enable Bugsnag in test environment (I think we use `RACK_ENV=test`, right?)

Reply:
I'm not sure about this. I don't think there's much value on enabling Bugsnag reporting from the test environment, so we would only use `BUGSNAG_ENABLED_STAGES` for `development`. So we would have an env variable to enable or disable bugsnag reporting, but we already have two variables for that: `CONFIG_HOPTOAD_SERVICE` and `CONFIG_HOPTOAD_API_KEY`

Reply:
Oh, OK... Thanks to your comment I've just figured out why setting `BUGSNAG_API_KEY` was not working for me :smiling_face_with_tear: To be honest, that's very confusing... we probably need to get rid of this over complex setup and just use code for BugSnag. But... in another PR :sweat_smile:

Reply:
Yeah, this come from before when we had Airbrake and Bugsnag, but now we only have Bugsnag we can remove `CONFIG_HOPTOAD_SERVICE` and `CONFIG_HOPTOAD_API_KEY` can be renamed to `CONFIG_BUGSNAG_API_KEY` or something similar. Maybe I can do it in this very PR. Let me check.

Reply:
After taking a look, I think it's better to not rename the variables, because it's a breaking change and it doesn't fix anything or adds a new feature. Not worth it.

About getting rid of this complex setup. The problem is we have a complex system to support several error monitoring services. Though we only support Bugsnag. So IMO we should decide if we want to keep this system just in case we need to support another service in the future or we go with Bugsnag and simplify this a lot. I would just keep the current system, so we avoid breaking changes and have the option to add new services in the future, for instance if a client requests it.

Reply:
If it ain't broken, don't fix it..